1.8/usage/service-discovery/haproxy-adminrouter.md

```diff
@@ -0,0 +1,107 @@
+---
+post_title: Configuring HAProxy in Front of Admin Router
+nav_title: HAProxy in Front of Admin Router
+menu_order: 400
+---
+
+You can use HAProxy to set up an HTTP proxy in front of the DC/OS [Admin Router](/docs/1.8/overview/concepts/#adminrouter).
```

Comment:
@jgehrcke Thanks! I was talking to Albert about this because I was not sure where one would insert the certificate that is signed by the valid certificate authority...I didn't see it here. Do you think it might be good to add? He was saying there were two use cases so far that we have run into: to run on a custom port and to use a valid cert. We could possibly mention those as "for example," I don't know if you might be amenable. Just as a couple of examples to help people who might not be sure, let me know your thoughts.